extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -365,18 +380,19 @@ export class AmpStoryEmbeddedComponent {
    */
   closeFocusedState_() {
     this.clearTooltip_();
-    this.setState_(ComponentState.HIDDEN, null /** target */);
+    this.setState_(EmbeddedComponentState.HIDDEN, null /** component */);
```

Comment:
This should be done from `onEmbeddedComponentUpdate_` in reaction of the action you're dispatching on the next line (is it called twice now?)

Reply:
Done.